.github/workflows/benchmarks.yml

```diff
@@ -62,12 +62,28 @@ jobs:
           path: ${{matrix.test_script}}.csv
 
   process_data:
+    # Only run on main repository
+    # Scheduled workflows do not have information about fork status, hence the hardcoded check
+    if: github.repository == 'G-Research/NuPerfMonitor'
+    environment: updater
     runs-on: ubuntu-latest
-    needs: [benchmark_dotnet_windows, benchmark_dotnet_linux]
+    needs: [benchmarks_windows, benchmarks_linux]
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-        
+
+      - name: Create access token
+        id: token
+        env:
+          APP_ID: ${{ secrets.APP_ID }}
+          APP_PRIVATE_KEY: ${{ secrets.APP_PRIVATE_KEY }}
+        run: |
+          jwt=$(step crypto jwt sign --key /dev/fd/3 --issuer $APP_ID --expiration $(date -d +5min +%s) --subtle 3<<< $APP_PRIVATE_KEY)
```

Comment:
You would also need to install `step` before you can use it. I believe it's not installed by default on the GitHub runners.

Reply:
Ah I missed that, done.